docs/README.md

```diff
@@ -34,5 +34,5 @@ curl -X GET \
 ```
 
 ::: tip VERSION NOTE
-Arable API 2.3.0 is now commercially released. See our changelog for release changes.
+Arable API 3.0.0 is now commercially released. See our changelog for release changes.
```

Comment:
I think we're on 3.1.0: https://github.com/Arable/api-user/releases/tag/v3.1.0

Which makes me think - can we use this opportunity to separate the public facing version from the version in api-user? Or otherwise at least make this text here automatically update to the same version that's in api-user?

Reply:
I think this is the line that would need to change: https://github.com/Arable/api-user/blob/develop/arable_api_user/v2/api_user.py#L173 to something like version='3.0.0'

Reply:
An endpoint in api-user returns api meta data would be a better solution to avoid updating the version in every release.